src/hooks/wallets/wallets.ts

```diff
@@ -45,6 +46,7 @@ const WALLET_MODULES: { [key in WALLET_KEYS]: (chain: ChainInfo) => WalletInit }
   [WALLET_KEYS.LEDGER]: () => ledgerModule() as WalletInit,
   [WALLET_KEYS.TREZOR]: () => trezorModule({ appUrl: TREZOR_APP_URL, email: TREZOR_EMAIL }) as WalletInit,
   [WALLET_KEYS.KEYSTONE]: () => keystoneModule() as WalletInit,
+  [WALLET_KEYS.PK]: (chain) => pkModule(chain.chainId, chain.rpcUri) as WalletInit,
```

Comment:
I think we ensure that this wallet can only be enabled when `IS_PRODUCTION` is `false` as it is a tool for our e2e tests.

Reply:
To enable it, you'd have to go to the production config service, create a PK wallet, and enable it on a chain. Don't think this can be done accidentally.